gqlalchemy/memgraph.py

```diff
@@ -327,6 +327,12 @@ def save_node(self, node: Node) -> Node:
         result = self._save_node_properties_on_disk(node, result)
         return result
 
+    def save_nodes(self, nodes: List[Node]) -> None:
+        """Saves a list of nodes to Memgraph."""
+        for i in range(len(nodes)):
+            saved_node = self.save_node(nodes[i])
+            nodes[i]._id = saved_node._id
+
```

Comment:
Method [`save_node()`](https://github.com/memgraph/gqlalchemy/blob/ad9a4055698928c51de405cf20228e52b158f48a/gqlalchemy/memgraph.py#L301) returns saved node. Here nothing is returned. That's fine if we decide to do it like that, but it's a bit inconsistent with the `save_node()` method.